evap/staff/urls.py

```diff
@@ -84,6 +84,8 @@
     path("faq/", views.faq_index, name="faq_index"),
     path("faq/<int:section_id>", views.faq_section, name="faq_section"),
 
+    path("infotexts/", views.infotexts, name="infotexts"),
```

Comment:
I wonder if it should be `/infotexts/edit` for, we are not super consistent with this, but it feels right to me. @He3lixxx ?

Reply:
I would not add the "edit". Like with text answer warnings, degrees and course types, we only have one page acting as overview list and edit page for these things, so there is no need to be more specific.